scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -1056,6 +1082,9 @@ public void addRecordsForDeferredProcessing(Record incomingMessage) {
 	public List<Record> takeDeferredRecords() {
 		List<Record> records = new ArrayList<Record>(deferredRecords);
 		deferredRecords.clear();
+		for (Record record : records) {
+			deferredRecordsSize -= record.size();
```

Comment:
This should not happened if there is not a bug but should we ensure that `deferredRecordsSize` is not less than 0 ?

Reply:
OK

Reply:
I'm not sure my proposition above was a good one, so I'm ok if we don't add anything about that.
But as you said OK, I search how you implement this and I didn't find that.
did I missed something ?

Reply:
Uups, just missed it.

Reply:
Now, done.